tests/test_using.py

```diff
@@ -2009,6 +2010,134 @@ class Meta:
         self.assertEqual(2, obj.two)
         self.assertEqual(((), {'one': 2, 'two': 3}), obj.extra)
 
+    def test_related_factory_list_of_varying_length(self):
+        # Create our list of expected "related object counts"
+        related_list_lengths = [5, 5, 4, 4, 3, 3, 2, 2, 1, 1]
+        RELATED_LIST_LENGTH = lambda: related_list_lengths.pop()
+        
+        class TestRelatedObject(object):
+            def __init__(self, obj=None, one=None, two=None):
+                # Mock out the 'List of Related Objects' generated by RelatedFactoryList
+                if hasattr(obj, 'related_list'):
+                    obj.related_list.append(self)
+                else:
+                    obj.related_list = [self]
+                self.one = one
+                self.two = two
+                self.three = obj
+
+        class TestRelatedObjectFactoryList(factory.Factory):
+            class Meta:
+                model = TestRelatedObject
+            one = 1
+            two = factory.LazyAttribute(lambda o: o.one + 1)
+
+        class TestObjectFactory(factory.Factory):
+            class Meta:
+                model = TestObject
+            one = 3
```

Comment:
these variable names are confusing when skimming the code... one what? At first glance I assume this is integers, so why is `one` set to `3`?

Is there a slightly-more verbose naming scheme that makes it a little more clear of the intent of these variables?